pyproject.toml

```diff
@@ -42,3 +42,19 @@ build-backend = "poetry.core.masonry.api"
 [[tool.poetry.source]]
 name = "pypi-public"
 url = "https://pypi.org/simple/"
+
+# Must be kept around for python 3.8 which requires poetry <= 1.8.5
```

Comment:
Hm, maybe we can transition from 3.8 in general? And support only [3.9; 3.12) range? https://devguide.python.org/versions/, 3.8 is not supported anymore.

If it is not quick, I am ok with submitting this PR but let's leave todo here then and let's do it asap because this duplication is not nice and clean.